ipyleaflet/external/here.py

```diff
@@ -25,6 +25,9 @@
 from ipyleaflet import TileLayer
 
 
+attribution = '&copy; <a href="https://here.com">HERE.com</a>'
```

Comment:
What is the reason for using a global variable now?

Reply:
Nothing special, a left-over from the original version. Will make it a fixed string.

Reply:
Done.